android/app/src/main/res/navigation/navigation.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    app:startDestination="@id/start_fragment">
```

Comment:
change this id to the id of the fragment that you want to start by default (so that you don't have to do all the navigation every time)

Reply:
e.g signup_to_volunteer_home

Reply:
e.g for request detail fragment, is to change `app:startDestination="@id/request_detail_fragment"` in `navigation.xml` and add a fake default, see below:

```
    @Module
    companion object {

        @Provides
        @JvmStatic
        @FragmentScope
        fun request(fragment: RequestDetailsFragment): Request {
            val request = fragment.arguments?.getSerializable("request")
            return if (request == null) fakeRequest else request as Request
        }
    }

val fakeRequest = Request(
    "masks",
    "The Royal London Hospital",
    "description",
    "Whitechapel Rd, Whitechapel, London E1 1FR, United Kingdom",
    "https://www.bartshealth.nhs.uk/media/images/versions/img94joktmu717322.jpg",
    1000
)
```